shelley/chain-and-ledger/executable-spec/src/LedgerState.hs

```diff
@@ -968,12 +960,12 @@ poolRewards
   -> Coin
   -> Coin
 poolRewards d_ sigma blocksN blocksTotal (Coin maxP) =
-  if intervalValue d_ < 0.8
-    then floor (p * fromIntegral maxP)
-    else 1
```

Comment:
this was a :bug:, the apparent performance ("p bar") should be 1 when `d` is at or above 0.8, _not_ the pool rewards.

Reply:
yes, exactly. Section 3.9.3 of the Delegation Design document recommends this, since the low fraction of blocks produced when d >= 0.8 means that the performance will be dominated by the randomness in the leader election. I thought the justification made as much sense in pool ranking as in rewards (both of which use the apparent performance).